Makefile.PL

```diff
@@ -8,6 +8,7 @@ use ExtUtils::MakeMaker;
 use File::Spec::Functions qw( catfile rel2abs );
 use Getopt::Long qw( GetOptionsFromArray );
 
+push @INC, '.';
 unless( require( catfile qw(inc IO Interactive Tiny.pm) ) ) {
```

Comment:
Surely, the right thing to do here would be to augment the call to "require" to start with "./", as modifying `@INC` for the entirety of Makefile.PL is not actually necessary here.